src/main/java/pl/printo3d/onedcutter/cutter1d/cutter/services/OrderService.java

```diff
@@ -64,10 +63,6 @@ public ResultModel makeOrder(OrderModel orderModel)
     this.setOrder(orderModel);
     /** END ZAPIS DO BAZY */
 
-    orderList.clearOrder();
```

Comment:
Tym clearem zabezpzieczasz się przed tym że w orderList będą jakieś stare dane z poprzednich zapytań ale to w sumie nie ma sensu bo po co mieć w ogóle takie pole